examples/secure_cloud_run_standalone/main.tf

```diff
@@ -84,6 +78,20 @@ module "secure_cloud_run" {
   env_vars                                = [{ name = "TEST", value = "true" }]
   domain                                  = var.domain
 
+  # If you are going to use secrets as volume uncomment this part of the code and fill with your values.
```

Comment:
I think this comment code can be removed, since this is a standalone example, where the harness is created and we are not creating the secret in the security project.